package-overrides/npm/jquery-countdown@2.2.0.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "jquery": ">=1.7"
```

Comment:
The best we have for this is simply `jquery: '*'`. This is tracked in https://github.com/jspm/jspm-cli/issues/1253.